packages/leavittbook/src/demos/leavitt-company-select/leavitt-company-select-playground.ts

```diff
@@ -0,0 +1,148 @@
+/* playground-fold */
```

Comment:
leavitt-company-select component needs 
`/**
       *  @ignore
       */`
above `this.dispatchEvent(new LeavittCompanySelectSelectedEvent(company));`

so the api-docs don't try to double document selected event